tests/test_user.py

```diff
@@ -1,45 +1,81 @@
-# Unit tests related to 'Users' (https://www.easypost.com/docs/api#users).
-
 import pytest
 
 import easypost
 
 
 @pytest.mark.vcr()
-def test_retrieve_me(prod_api_key):
+@pytest.mark.skip(reason="This endpoint returns the child user keys in plain text, do not run this test.")
+def test_user_create(prod_api_key):
+    user = easypost.User.create(name="Test User")
+
+    assert isinstance(user, easypost.User)
+    assert str.startswith(user.id, "user_")
+    assert user.name == "Test User"
+
+
+@pytest.mark.vcr()
+def test_user_retrieve(prod_api_key, child_user_id):
+    user = easypost.User.retrieve(child_user_id)
+
+    assert isinstance(user, easypost.User)
+    assert str.startswith(user.id, "user_")
+
+
+@pytest.mark.vcr()
+def test_user_retrieve_me(prod_api_key):
+    user = easypost.User.retrieve_me()
+
+    assert isinstance(user, easypost.User)
+    assert str.startswith(user.id, "user_")
+
+
+@pytest.mark.vcr()
+def test_user_update(prod_api_key):
     user = easypost.User.retrieve_me()
 
-    assert user.id is not None
+    test_phone = "5555555555"
+
+    user.phone = test_phone
+    user.save()
+
+    assert isinstance(user, easypost.User)
+    assert str.startswith(user.id, "user_")
+    assert user.phone == test_phone
 
 
 @pytest.mark.vcr()
-def test_child_user_create(prod_api_key):
-    # Create an address and then verify some fields to test whether it was created just fine.
-    child_user = easypost.User.create(name="Python All-Things-Testing")
+@pytest.mark.skip(
+    reason="Due to our inability to create child users securely,"
```

Comment:
You'll need a space after the comma or before the next `we`, when concatenating strings it'll slap them together without a space unless you add one yourself.

Reply:
thanks for letting me know that!